ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Query.hs

```diff
@@ -74,7 +90,9 @@ queryEncodeNodeToClient codecConfig queryVersion blockVersion (SomeSecond query)
     TopLevelQueryDisabled -> encodeBlockQuery
       (case query of
         BlockQuery blockQuery -> blockQuery
+        GetSystemStart -> error "TODO: handle not available in this version"
```

Comment:
Using `error` here is generally what we do in this function / other `SerialiseNodeToClient` instances.